### PR TITLE
[OMEVV]: Utils file 

### PR DESCRIPTION
**Pull request by ABHISHEK-SINHA10**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: plugins/module_utils/omevv.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+from ansible_collections.dellemc.openmanage.plugins.module_utils.rest_api import RestAPI, RestAnsibleModule
+from ansible.module_utils.common.parameters import env_fallback
+
+root_omevv_uri = "/omevv/GatewayService/v1"
+session_resource = {}
+
+
+class RestOMEVV(RestAPI):
+    def __init__(self, module_params, protocol="https",
```
Comment (rajshekarp87) [inline plugins/module_utils/omevv.py:41]: Kindly add comment for all methods

Comment (ABHISHEK-SINHA10) [reply plugins/module_utils/omevv.py:41]: Added doc in invoke_request, rest is constructor.

**Review by rajshekarp87 — DISMISSED** [review]

**Comment by sachin-apa** [comment]
run e2e

**Review by sachin-apa — COMMENTED** [review]
1. Lets check on the sonar, checkmarx and blackduck issues if any.
2. Lets make sure coverage via UT is >90% for all the files added.

**Comment by sachin-apa** [comment]
run e2e

**Review by rajshekarp87 — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Pull request merged**